src/main/java/uk/gov/ons/census/casesvc/logging/EventLogger.java

```diff
@@ -71,6 +73,30 @@ public void logRefusalEvent(
         eventMetaDataDateTime);
   }
 
+  public void logFulfilmentRequestedEvent(
+      Case caze,
+      UUID caseId,
+      String eventDescription,
+      EventType eventType,
+      FulfilmentRequestDTO payload,
+      EventDTO event) {
+
+    Event loggedEvent = new Event();
+    loggedEvent.setCaze(caze);
+    loggedEvent.setId(UUID.randomUUID());
+    loggedEvent.setCaseId(caseId);
+    loggedEvent.setEventDate(OffsetDateTime.now());
```

Comment:
Is it worth using the date from the event itself here?

Reply:
yep agreed, this should be the date contained in the event passed to us